lib/autorestPlugin/pluginHost.ts

```diff
@@ -19,21 +19,24 @@ const extension = new extensionBase.AutoRestExtension();
 const modelValidatorPluginName = "model-validator";
 const modelValidationCategory = "ExampleModelViolation";
 
-function FormattedOutput(channel, details, code, text, source) {
-  this.channel = channel;
-  this.details = details;
-  this.code = code;
-  this.text = text;
-  this.source = source;
+class FormattedOutput{
+  constructor(
+    public channel: any,
+    public details: any,
+    public code: any,
+    public text: any,
+    public source: any,
+    _?: any) {
```

Comment:
what is this for ?

Reply:
there's a call that uses more parameters than declared. It's definitely a bug that we need to fix. Ignore this for now :-)

Reply:
ok